src/entities/mellow/mellowLpRouter.ts

```diff
@@ -633,6 +634,91 @@ class MellowLpRouter {
       throw new Error('Unsucessful rollover confirmation.');
     }
   };
+
+  registerForAutoRollover = async (registration: boolean): Promise<ContractReceipt> => {
+    if (
+      isUndefined(this.readOnlyContracts) ||
```

Comment:
you can remove this as well this.readOnlyContracts

Reply:
addressed